### ISSUE TITLE
Future simplification="important" technique

### ISSUE THREAD

**Issue by yatil** [w3c/wcag#1282]
I see that in the Understanding of 3.2.7: Hidden Controls mentions a future sufficient technique of adding `simplification="important"` to indicate the control is important.

https://www.w3.org/WAI/WCAG22/Understanding/hidden-controls.html#sufficient

I don’t think there are implementations of that or specifications that would support that. Maybe rephrase to “Indicate programmatically that a control is important (future)”.

(I see in Audits that people copy and paste such pseudocode and it then shows up in reviews, basically doing nothing.)

**Comment by alastc** [comment]
HI @yatil,

I think that's editorial, so I merged that with #1616
